contracts/estate/EstateRegistry.sol

```diff
@@ -0,0 +1,236 @@
+pragma solidity ^0.4.22;
+
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+import "zeppelin-solidity/contracts/token/ERC721/ERC721Token.sol";
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+
+import "../metadata/MetadataHolderBase.sol";
+
+
+contract PingableDAR {
```

Comment:
If `EstateRegistry` is not a generic contract (`ComposableNFTs`), What do you think of using specific names as

```
PingableDAR ~ LandRegistry
dar ~ landRegistry
onlyDAR ~ onlyLandRegistry
```

@eordano  @abarmat  @NicoSantangelo

Reply:
I would like to know your thoughts on this too. I do like to be more specific if we don't really need to manage generic data. It makes the contract easier to read